htlcswitch/link.go

```diff
@@ -395,11 +402,12 @@ func NewChannelLink(cfg ChannelLinkConfig,
 		channel:     channel,
 		shortChanID: channel.ShortChanID(),
 		// TODO(roasbeef): just do reserve here?
-		htlcUpdates: make(chan *contractcourt.ContractUpdate),
-		hodlMap:     make(map[channeldb.CircuitKey]hodlHtlc),
-		hodlQueue:   queue.NewConcurrentQueue(10),
-		log:         build.NewPrefixLog(logPrefix, log),
-		quit:        make(chan struct{}),
+		htlcUpdates:  make(chan *contractcourt.ContractUpdate),
+		hodlMap:      make(map[channeldb.CircuitKey]hodlHtlc),
+		hodlQueue:    queue.NewConcurrentQueue(10),
+		log:          build.NewPrefixLog(logPrefix, log),
+		quit:         make(chan struct{}),
+		downstreamIn: make(chan *downstreamMsg),
```

Comment:
group with other downstream chnanel

Reply:
the other downstream chan isn't initialized here